app/filtering.js

```diff
@@ -31,6 +31,7 @@ const app = electron.app
 const uuid = require('node-uuid')
 const path = require('path')
 const getOrigin = require('../js/state/siteUtil').getOrigin
+const {adBlockResourceName} = require('./adBlock')
```

Comment:
Small style question- our project has a mix of:
- destructuring assignments with and without spaces around the item(s) (see line 34)
- including using require and then referencing the method name (see line 33)

Is there a preferred syntax?